src/components/common/StaffPayment.vue

```diff
@@ -4,17 +4,89 @@
       <h2>3. Staff Payment</h2>
     </header>
 
-    <div id="staff-payment-container">
-      Under construction!
-    </div>
+    <staff-payment-component
+      :staffPaymentData="getStaffPayment"
+      @update:staffPaymentData="onStaffPaymentData"
+      @valid="emitValid"/>
   </section>
 </template>
 
 <script lang="ts">
-import { Component, Vue } from 'vue-property-decorator'
+import { Component, Emit, Vue, Watch } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
 
-@Component({})
+// Components
+import { StaffPayment as StaffPaymentComponent } from '@bcrs-shared-components/staff-payment'
+
+// Mixins, Interfaces and Enums
+import { FilingTemplateMixin } from '@/mixins'
+import { ActionBindingIF } from '@/interfaces'
+
+import { StaffPaymentIF } from '@bcrs-shared-components/interfaces'
+import { StaffPaymentOptions } from '@bcrs-shared-components/enums'
```

Comment:
Very cool, would love to see universal interfaces/enums across all our UI's one day.